idaes_examples/notebooks/docs/unit_models/custom_unit_models/liquid_liquid_extractor.py

```diff
@@ -0,0 +1,347 @@
+#################################################################################
```

Comment:
Would these be best moved to a sub-folder to keep them from cluttering the top level folder (and maybe make them easier to find)?